src/services/functionAppService.test.ts

```diff
@@ -107,8 +107,8 @@ describe("Function App Service", () => {
 
   it("gets master key", async () => {
     const service = createService();
-    const masterKey = await service.getMasterKey();
-    expect(masterKey).toEqual(masterKey);
+    const result = await service.getMasterKey();
```

Comment:
nit: I would call this `actualMasterKey`

Reply:
will do